front/app/services/customPages.ts

```diff
@@ -126,70 +126,36 @@ export function listCustomPages(streamParams: IStreamParams | null = null) {
   });
 }
 
-// The following types and resources were moved from old "pages" code.
-// Mainly they refer to pages that have special properties, like
-// the FAQ/about pages, and terms/conditions and privacy policies
-
-// These types refer to the "code" attribute of a page.
-// The 'standard page' distinction is only relevant for non-commercial
-// customers: they can edit the content of these pages, but nothing else.
-// For commercial customers, these behave as 'custom' pages.
-type TStandardPage = 'about';
-
-// Policy pages of which only the content can be edited
-// in 'policy' tab in settings (both for non-commercial and
-// commercial customers). Their codes are the same as their slugs.
-export type TPolicyPage = 'terms-and-conditions' | 'privacy-policy';
-
-export const POLICY_PAGES: TPolicyPage[] = [
-  'terms-and-conditions',
-  'privacy-policy',
-];
-
-export enum POLICY_PAGE {
+enum POLICY_PAGE {
   termsAndConditions = 'terms-and-conditions',
   privacyPolicy = 'privacy-policy',
 }
 
+export const POLICY_PAGES: TPolicyPage[] = [
+  POLICY_PAGE.termsAndConditions,
+  POLICY_PAGE.privacyPolicy,
+];
+
 export function isPolicyPageSlug(slug: string): slug is TPolicyPage {
   const termsAndConditionsSlug: TPolicyPage = POLICY_PAGE.termsAndConditions;
   const privacyPolicySlug: TPolicyPage = POLICY_PAGE.privacyPolicy;
 
   return slug === termsAndConditionsSlug || slug === privacyPolicySlug;
 }
 
-// Hardcoded pages don't actually exist in the database-
-// their codes are the same as their slugs, which are used to render
-// the footer. The slugs link to hard-coded components, see app/routes.ts.
-type THardcodedPage = 'cookie-policy' | 'accessibility-statement';
-
-// Pages in the footer.
-export type TFooterPage = TPolicyPage | THardcodedPage;
-
-export const FOOTER_PAGES: TFooterPage[] = [
-  'terms-and-conditions',
-  'privacy-policy',
-  'cookie-policy',
-  'accessibility-statement',
-];
-
-// Pages that exist in the static_pages database,
-// but do not have a corresponding navbar item (with the exception of 'faq').
-// Their slugs and titles cannot be changed. Their
-// codes are the same as their slugs.
-// proposals here is the proposals about page (/pages/proposals),
-// not the proposals index page (/initiatives).
-type TFixedPage = TPolicyPage | 'faq' | 'proposals';
-
-export const FIXED_PAGES: TFixedPage[] = [
-  'terms-and-conditions',
-  'privacy-policy',
-  'proposals',
-];
```

Comment:
The concept of a "fixed" page doesn't necessarily make sense in customPages types. The only reason they are fixed is because we are linking to them internally in the codebase, so changing them would break links. Therefore I moved that explanation to `front/app/containers/Admin/pagesAndMenu/containers/CustomPages/Edit/Settings/index.tsx` above, where we decided for which pages to render the slug input edit field.